docs/guide/setting-up.md

```diff
@@ -107,4 +107,4 @@ If you are starting the site in a production environment or do not want the proc
 
 ## Next steps
 
-Now that Apostrophe is installed, you're ready to start building. Check out the essentials [guide](/guide/) to learn about essential features or read more about Apostrophe's inner workings in the [technical overview](/guide/technical-overview.md).
+Now that Apostrophe is installed, you're ready to start building. Check out the [guide](/guide/modules.html) to learn about essential features with plenty of code examples. If you are looking to use Apostrophe API as a headless CMS or explore Apostrophe's inner workings peruse the [reference guide](/reference/glossary.md). 
```

Comment:
* Link is to the glossary, does that make sense?
* For the headless CMS use case we really need a landing page that gives you a quick cheat sheet on creating piece types and then links you to the REST API documentation. lacking that for the moment, maybe we don't mention it here yet.

Reply:
For the first point, I'm only linking there because it is the first page of our reference section. Maybe the answer is that the reference section should also be reshuffled? Glossary moves to the end of the list with the API routes, second to last, and the remainder of the sections in the same order?

For the second point, reference to the headless CMS removed and ticket made for creating that page.